http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -411,6 +413,11 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // Sam configs
         raw"sam.url=${config.samConfig.server}",
 
+        // Enabled services configs
```

Comment:
Do we need to also include `cbasUI` here? (Perhaps it is tied 1<>1 with Cbas being present)?

Reply:
Oh -- good catch! Yeah will fix that